forest/sycamore/common.py

```diff
@@ -532,6 +539,10 @@ def append_from_answers(
             The last date of the survey data, in YYYY-MM-DD format
         config_path(str):
             Filepath to survey config file (downloaded from Beiwe website)
+        history_path: Path to survey history file. If this is included, the
```

Comment:
Either style is acceptable but you should continue putting arg names on a separate line for consistency.